cleaner.go

```diff
@@ -58,6 +187,10 @@ func NewCleaner(kubeConfigContent []byte) (*Cleaner, error) {
 		}
 	}
 
+	restConfig.Burst = 100
+	restConfig.QPS = 500
+	restConfig.RateLimiter = nil
```

Comment:
the job was getting client-sire ratelimitted which was prolonging the cleanup to more than an hour.